resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/scheduler/VolcanoFeatureStep.scala

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Please add this directory `org/apache/spark/deploy/k8s/features/scheduler` if and if only `-Pvolcano` is enabled.